client/src/components/_common/SectionHeader/SectionHeader.module.css

```diff
@@ -16,7 +16,7 @@
     var(--global-color-primary--dark);
 }
 /* FP-546: Remove this because <Button> handles font-size */
-.root > a:global(.btn) {
+.root > a:global(.btn-secondary) {
```

Comment:
Allocations had a `.btn` in it's `<SectionHeader>` which became small—not intended.

Also for FP-1491 (#598), Design said no small primary buttons.